tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 2
+Standard output:
+lib/ContextModule.js(352,32): error TS2345: Argument of type 'Pick<ContextOptions & ContextModuleOptionsExtras, "include" | "regExp" | "chunkName" | "referencedExports" | "mode" | "recursive" | ... 6 more ... | "resourceQuery">' is not assignable to parameter of type 'ContextOptions & ContextModuleOptionsExtras'.
+  Property 'resolveOptions' is missing in type 'Pick<ContextOptions & ContextModuleOptionsExtras, "include" | "regExp" | "chunkName" | "referencedExports" | "mode" | "recursive" | ... 6 more ... | "resourceQuery">' but required in type 'ContextModuleOptionsExtras'.
```

Comment:
I'm not sure why this worked before, but the code tries to assign `Omit<X, "p">` to `X` and that *definitely* means you should get a "missing 'p'" error.

Reply:
Previously, the function which was called here was typed as `any`. I think it's because it was assigned `any` outside the constructor. Regardless, the new type is correct.